### PR TITLE
Todo list implementation using Javascript and Webpack.

### PR DESCRIPTION
**Pull request by Abhigyan001**
- This is a To Do List created using Javascript and Webpack. 
- An user can add multiple projects and add multiple tasks under each project. 
- User can add task title, task description, duedate and set the priority of the tasks.
- User can also update the tasks as well as remove it. 
- The project can be deleted as well.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "todo-list",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```
Comment (addod19) [inline package.json]: Let us kindly improve our script and make it better so we can run commands like `npm run build`, `npm run dev` and many more
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```

File: package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    "babel-eslint": "^10.1.0",
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.1.0",
+    "eslint-plugin-import": "^2.20.2",
+    "stylelint": "^13.3.3",
+    "stylelint-config-standard": "^20.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.17.2",
+    "webpack": "^5.10.0",
+    "webpack-cli": "^4.2.0"
```
Comment (addod19) [inline package.json:32]: We are missing the `webpack-dev-server` in the `devDependencies`. Let us add it

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @Abhigyan001 and @ClaytonSiby, great job on this project :+1: Most of the functionalities were implemented :slightly_smiling_face: I would highlight a few to address. Kindly check the comments below:
- The `delete` functionality is missing from the project, let us add it. A user must be able to delete a todo :smiley: 
del 
- There should be a default project :+1: Currently, we do not have it
def

[Optional]
- The `edit` button overlaps each other as in the screenshot below. Let us make it better
op

Once these are fixed, kindly submit another code review :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

File: package.json (hunk 1 above)
Comment (Abhigyan001) [reply package.json]: added

**Comment by Abhigyan001** [comment]
A user can delete the task by clicking on the radio button to mark it as complete and then by clicking on the button 'Clear Completed Task'. The "Edit" button has been redesigned and a "Default" project has been added.

**Review by wathigo — APPROVED** [review]
### Approved :heavy_check_mark: 
Great job working on the suggested changes.

**You may move on.**

_Have a great weekend._
**Simon Wathigo.**

**Pull request merged**